Topics/Tech_Stacks/TypeScript.md

```diff
@@ -57,6 +57,21 @@ Angular comes with TypeScript enabled by default. In fact, Angular is built enti
 
     `ng new my-app`'
 
+### Vue
+
+To use TypeScript in your Vue 3 project using `create-vue`, which is the official Vue project scaffolding tool, follow the steps below:
+
+1. Open a terminal window, navigate to the location where the project will be created. 
+
+2. Run the command `npm create vue@latest` to initiate the project creation. (Note: if you do not have Node.js installed on your machine, follow the guide in the sections above to install it first before creating the Vue project)
```

Comment:
Consider inserting specially-formatted text (or screenshots from the terminal) at each step, so the user knows exactly what to do.

Reply:
Thanks, I will include screenshots of the terminal to show the exact prompts to follow.